integration/README.md

```diff
@@ -0,0 +1,118 @@
+```
+Mounting of the directories `-v /var/run/docker.sock:/var/run/docker.sock -v /proc:/proc_host -v /usr/bin/docker:/usr/local/bin/docker -v /var/lib/docker:/var/lib/docker` and PROCFS_MOUNT environment variable `-e PROCFS_MOUNT=/proc_host` is needed for docker collector to gather metrics from the host.
+```sh
+$ docker exec -ti snap bash
+```
+Now that you are inside the container with running Snap daemon, you may run command to list the plugins
+```sh
+$ snaptel plugin list
+``` 
+The output should be `No plugins found. Have you loaded a plugin?` as there are no plugins loaded yet. Let's download some...
+
+```sh
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-collector-docker/releases/download/5/snap-plugin-collector-docker_linux_x86_64" -o snap-plugin-collector-docker
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64" -o snap-plugin-publisher-file
+```
+...and load them.
+```sh
+$ snaptel plugin load snap-plugin-collector-docker
+$ snaptel plugin load snap-plugin-publisher-file
+```
+Now, running the command
+```sh
+$ snaptel plugin list
+``` 
+will print information about the two loaded plugins. To start the collection of metrics you can create task.
+```sh
+$ curl -sO https://raw.githubusercontent.com/intelsdi-x/snap-plugin-collector-docker/master/examples/tasks/docker-file.json
+$ snaptel task create -t ./docker-file.json
+``` 
+Command
+```sh
+$ snaptel task list
+``` 
+will provide information whether the metrics collection was successfull or not.
+
+### Running Snap on Kubernetes cluster
+
+To run Snap in Kubernetes pods create daemonset from manifest file `snap.yaml`.
+```sh
+$ kubectl create -f kubesnap/integrations/k8s/snap.yaml
+```
+Verify that pods have been created.
+```sh
+$ kubectl get pods --namespace=kube-system
+```
+Log into the one of pods using the pod name returned by `kubectl get pods` command.
+```sh
+$ kubectl exec -ti snap-xxxxx bash --namespace=kube-system
+```
+Now that you are inside the pod with running Snap daemon, you may run command to list the plugins
+```sh
+$ snaptel plugin list
+``` 
+The output should be `No plugins found. Have you loaded a plugin?` as there are no plugins loaded yet. Let's download some...
+
+```sh
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-collector-docker/releases/download/5/snap-plugin-collector-docker_linux_x86_64" -o snap-plugin-collector-docker
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-publisher-file/releases/download/2/snap-plugin-publisher-file_linux_x86_64" -o snap-plugin-publisher-file
+$ curl -fsL "https://github.com/intelsdi-x/snap-plugin-publisher-heapster/releases/download/1/snap-plugin-publisher-heapster_linux_x86_64" -o snap-plugin-publisher-heapster
+```
```

Comment:
To use this plugin you need a heapster - I'd propose to leave file publisher here and add separate sub-section in K8s section describing how to use it with heapster publisher.